### ISSUE TITLE
Auto close option

### ISSUE THREAD

**Issue by sllloyd** [crossxcell99/dtsel#11]
Many date pickers have an autoclose feature that hides the picker once you have chosen a date. I couldn't work out how to do this properly but the following seems to work:

In function DTS defaultConfig add:
            autoClose: false

In DTBox.prototype.onDateSelected add this at the end:
if (this.settings.config.autoClose){
    var blur = new Event('blur');
    this.elem.dispatchEvent(blur); 
}

There is probably a better way.  Cheers

**Comment by lugui20** [comment]
I agree with you that would be nice an auto close option. I solved this by adding  `this.cancelBlur -= 1;` inside function `onDateSelected`, just after [line 525](https://github.com/crossxcell99/dtsel/blob/master/dtsel.js#L525).
